aspnetcore/diagnostics/asp0028.md

```diff
@@ -17,11 +17,17 @@ uid: diagnostics/asp0028
 
 ## Cause
 
-On the server machine that supports IPv6, listening to `Any`, rather than `IPv6Any` will either not work or be slower than necessary, because of the [underlying System types implementation](https://github.com/dotnet/runtime/issues/82404).
+[IPv6Any](/dotnet/api/system.net.ipaddress.ipv6any) is preferred to [Any](/dotnet/api/system.net.ipaddress.any) because `Any` is less performant than `IPv6Any`. In some cases, `Any` may not work at all. `Any` has performance problems due to the [underlying System types implementation](https://github.com/dotnet/runtime/issues/82404).
```

Comment:
phrasing `has performance problems` is not what I wanted to say - it raises a question of why not fix the System types then? I wanted to avoid this

Reply:
> phrasing `has performance problems` is not what I wanted to say - it raises a question of why not fix the System types then? I wanted to avoid this

I've reverted to your wording with my wording in comments below. I don't see the difference between ***slower than necessary*** and less performant.